Domains/3-DisciplineOther/Hydraulics/SewerHydraulicAnalysis.ecschema.xml

```diff
@@ -172,6 +172,22 @@
 		</Target>
 	</ECRelationshipClass>
 
+	<ECEntityClass typeName="Catchment" modifier="Sealed" displayLabel="Rainfall Set">
```

Comment:
"displayLabel" attribute needs to be updated.

Reply:
derp - the fix should now be published